packages/nx-playwright/README.md

```diff
@@ -17,17 +17,25 @@ yarn playwright install --with-deps
 yarn nx e2e <APP-NAME>-e2e
 ```
 
-## Execution Flags
-
-`nx-playwright` has some flags that you can utilize at execution time
-
-- `--browser=BROWSER_TYPE`: allowed browser types being `chromium`, `firefox` or `webkit` (or an `all` type to execute against all 3 types)
-- `--format=FORMAT_TYPE`: this allows values such as `json` or `html`
-- `--headed`: launches the browsers in non-headless mode
-- `--skipServe`: skips the execution of a devServer
-- `--timeout=TIMEOUT_IN_MS`: adds a timeout for your tests
+### Options
+
+The following options can be passed as the `"options"` field in the `"e2e"` target in project.json:
```

Comment:
```suggestion
The following options can be passed as values for `e2e.options` in `project.json`:
```

Reply:
If you implement the suggestion below (ie, linking directly to `schema.d.ts`), you'll need to tweak the suggested changes above.